Contents/Code/__init__.py

```diff
@@ -381,7 +394,7 @@ def Update(self, metadata, media, lang, force, movie):
       if list_eps !="":  Log.Debug("List_eps: %s" % str(sorted(list_eps)))
       Log.Debug("TVDB table: '%s'" % str(tvdb_table))
       
-    elif metadata_id_source == "anidb": 
+    elif metadata_id_source.startswith("anidb"):
```

Comment:
This is not required as 'anidb2' exists only in the scanner. The scanner converts the 'anidb2-xxx' to its 'tvdb-xxx' mapped id.